config/bootstrap.php

```diff
@@ -43,27 +42,6 @@
     ]);
 }
 
-Router::plugin('DebugKit', function ($routes) {
```

Comment:
I think my above comment addresses this as well. Unfortunately it doesn't seem possible right now :( I'm not stoked on the idea of breaking everyone's debug env, but any `Router::reload()` in a debug env currently breaks debug kit. The `Plugin::bootstrap('DebugKit')` workaround I've been using *works* but I foresee all sorts of potential problems.